ports/stm32/mpu.h

```diff
@@ -94,6 +95,18 @@
         | MPU_REGION_ENABLE << MPU_RASR_ENABLE_Pos \
     )
 
+#define MPU_CONFIG_SHARED_NCACHE(size) ( \
```

Comment:
I suggest calling this `MPU_CONFIG_SHARED_UNCACHED`, to match other names here.

Reply:
I actually added the first one, but other code was merged first. I used `NCACHE` so we can then add `NSHARED_NCACHED_NBUFFERED` if needed, just a single letter to negate configs, vs `NOTSHARED_UNCACHED`. I'll change it anyway.